plugins/modules/cloud/digital_ocean/digital_ocean_droplet.py

```diff
@@ -54,6 +54,11 @@
      - add an additional, private network interface to droplet for inter-droplet communication.
     default: False
     type: bool
+  vpc_uuid:
+    description:
+     - A string specifying the UUID of the VPC to which the Droplet will be assigned. If excluded, Droplet will be
+       assigned to the account's default VPC for the region.
+    required: False
```

Comment:
```suggestion
    type: str
```

Reply:
`required: False` does not need to be specified (only `required: True`). But `type` must always be specified.

The sanity checks don't complain here because there are a lot of ignore.txt entries (see tests/sanity/ignore-2.10.txt). Dear maintainers: could you please make the correct changes and remove these ignore.txt entries?